_automating-configurations/default-use-cases.md

```diff
@@ -0,0 +1,200 @@
+| semantic_search_with_cohere_embedding                | Same as above use case but without any query enricher processor setup                                                                                                                                                                                                             | API Key         |[pre-set-defaults-file](https://github.com/opensearch-project/flow-framework/blob/2.13/src/main/resources/defaults/bedrock-titan-embedding-defaults.json)                  |
+| multi_modal_search                                   | Sets up an ingest pipeline with a text_image_embedding processor and a matching KNN index for multimodal search. User should provide a model ID                                                                                                                                   | model_id        |[pre-set-defaults-file](https://github.com/opensearch-project/flow-framework/blob/2.13/src/main/resources/defaults/bedrock-titan-embedding-defaults.json)                  |
+| multi_modal_search_with_bedrock_titan_multi_modal    | Deploys an Amazon Bedrock multimodal model and sets up an ingest pipeline with a text_image_embedding processor and a matching KNN index for multimodal search. User should provide there AWS Credentials                                                                         | AWS Credentials |[pre-set-defaults-file](https://github.com/opensearch-project/flow-framework/blob/2.13/src/main/resources/defaults/bedrock-titan-embedding-defaults.json)                  |
+| hybrid_search                                        | Sets up hybrid search by creating an ingest pipeline, a KNN index and a search pipeline with a normalization processor for appropriate usage. User should provide a model_id                                                                                                      | model_id        |[pre-set-defaults-file](https://github.com/opensearch-project/flow-framework/blob/2.13/src/main/resources/defaults/bedrock-titan-embedding-defaults.json)                  |
+| conversational_search_with_llm_deploy                | Deploys an LLM model (defaulted to cohere) and sets up a search pipeline with a retrieval_augmented_generation processor for use.                                                                                                                                                 | API Key         |[pre-set-defaults-file](https://github.com/opensearch-project/flow-framework/blob/2.13/src/main/resources/defaults/bedrock-titan-embedding-defaults.json)                  |
+
+
+### Addtional Examples
```

Comment:
I'm not sure this section is needed if we have the detailed example above. Or at the least, examples should be grouped in the same general section, under a parent section.

Reply:
@kolchfa-aws do you have any thoughts?